guide_drafts/trait_derives.md

```diff
@@ -0,0 +1,954 @@
+Deriving Traits in Depth
+===============
+
+Part of what makes Diesel's query builder so powerful is
+its ability to assist writing safe SQL queries in Rust.
+Diesel enables this level of safety through implementing 
+a series of traits on your structs.
+Writing these implementations by hand can be very laborious,
+so Diesel offers custom derives.
```

Comment:
Every one of these three sentences has 'Diesel' in it. Maybe use 'it' in one of them?